docs/apache-airflow/concepts/dynamic-task-mapping.rst

```diff
@@ -180,17 +180,19 @@ It is possible to use ``partial`` and ``expand`` with classic style operators as
 Mapping over result of classic operators
 ----------------------------------------
 
-If you want to map over the result of a classic operator you will need to create an ``XComArg`` object manually.
+If you want to map over the result of a classic operator, you should explicitly reference the *output*, instead of the operator itself.
 
 .. code-block:: python
 
-    from airflow import XComArg
+    # Create a list of data inputs.
+    extract = ExtractOperator(task_id="extract")
 
-    task = MyOperator(task_id="source")
+    # Expand the operator to transform each input.
+    transform = TransformOperator.partial(task_id="transform").expand(input=extract.output)
 
-    downstream = MyOperator2.partial(task_id="consumer").expand(input=XComArg(task))
+    # Collect the transformed inputs, expand the operator to load each one of them to the target.
+    load = LoadOperator(task_id="load").expand(input=transform.output)
```

Comment:
`.partial` is missing, I don't think you can `expand` after you have already instantiated your operator
```suggestion
    load = LoadOperator.partial(task_id="load").expand(input=transform.output)
```